### ISSUE TITLE
"Java Projects" View only showing after reinstalling extension

### ISSUE THREAD

**Issue by FelipeKobra** [microsoft/vscode-java-dependency#846]
When I first installed the extension, the View ("Java Projects") appeared to me on the Explorer section, and, even now, it only shows if I reinstall it, but, if I reload VSCode, it disappears again and I can't even see it after clicking on the `...` on the Explorer.

All my extensions are on the last version, and so is VSCode.

- Windows 11
- VSCode 1.91.0
- Project Manager for Java 0.23.7

Java Version
![image](https://github.com/microsoft/vscode-java-dependency/assets/139709626/fd95d98f-c7d6-4fe8-9da3-1c2bb07b6674)

When I open a `.java` file it shows that is is loading, I can even see it build
![image](https://github.com/microsoft/vscode-java-dependency/assets/139709626/66a83eaa-0af0-4451-86f8-ee86316901f3)

**Comment by jdneo** [comment]
Do you mean each time you have to reinstall the extension to see the Java Projects explorer? Which extension did you reinstall?

What if you trigger `Focus on Java Projects View` from command palette?

**Comment by FelipeKobra** [comment]
> Do you mean each time you have to reinstall the extension to see the Java Projects explorer? Which extension did you reinstall?

Yeah friend, the `Project Manager for Java` extension.

Like, if I reinstall it and restart extensions it shows normally on the Explorer tab, but, if I exit and open VSCode again , or just reload the window, it just disappears, and I can't select it again on Explorer, even after opening the `...`
![image](https://github.com/microsoft/vscode-java-dependency/assets/139709626/e4bfb425-12a6-4f2a-8217-d537c0e7c651)

> What if you trigger Focus on Java Projects View from command palette?

This works! But I can see it only on the right side of the screen, and not on the Explorer tab, like where the `OUTLINE` and `TIMELINE` are

![yeah](https://github.com/microsoft/vscode-java-dependency/assets/139709626/842be81b-4c1b-4ce8-a8f2-64a47c077122)

![image](https://github.com/microsoft/vscode-java-dependency/assets/139709626/0dc2950b-6ae5-4108-b53f-d104dafbb8e8)

**Comment by jdneo** [comment]
Ok, this explorer is draggable, you can press down your mouse on the 
![image](https://github.com/microsoft/vscode-java-dependency/assets/6193897/576be066-765a-4db6-bd24-2a848de19eb0)
 And then, drag it into wherever you want.

**Comment by FelipeKobra** [comment]
Thanks friend! Now even when I reload the window it stays there.

Is this a common issue the `Java Projects` don't start on the Explorer tab after reloading the window? Because for me it happened some times using this extension

**Comment by jdneo** [comment]
It may related with the profile you are using. I'm not sure. But anyway, the location for all the explorers are customizable.

See: https://code.visualstudio.com/docs/editor/profiles

**Comment by Stargazer1209** [comment]
Thankfully this is also my problem, and now it has been solved.
But I would like to add more information on this topic, if helpful.

I encountered this problem right after a recent Github Copilot related update which moves the default chat panel position from explorer side to the right side of the screen (should it be called secondary sidebar, I guess?)
- However, after the update, copilot give a clear warning at the original position that it has been moved to a new place, while java project view 
 ( and maven view) does not give any warnings or tips. 
- What's worse, the buttons of secondary sidebar on the top side are way too small compared to those on the left of primary sidebar, which makes it difficult to notice copilot is not the only new member of this place.

And there is another problem: Focus on Java Projects View from command palette got a weird prefix as shown below:
![Image](https://github.com/user-attachments/assets/241aed63-580d-4990-b363-c286c60eab87)

if necessary, I would open a new issue for this, just let me know.
